.eslintrc.json

```diff
@@ -1,21 +1,16 @@
 {
-    "env": {
-        "browser": true,
-        "es6": true,
-        "node": true,
-        "mocha": true
-    },
-    "extends": [
-        "airbnb-base", "prettier", "plugin:node/recommended"
-    ],
-    "globals": {
-        "Atomics": "readonly",
-        "SharedArrayBuffer": "readonly"
-    },
-    "parserOptions": {
-        "ecmaVersion": 2018,
-        "sourceType": "module"
-    },
-    "rules": {
-    }
-}
\ No newline at end of file
+  "env": {
+    "es6": true,
+    "node": true
+  },
+  // "extends": ["airbnb-base"],
```

Comment:
Why aren't you following Airbnb style guide?

Reply:
as I can see it is commentend

Reply:
I had commented it to avoid errors, but it's not in the requirements I wanted to first push the changes I made so far, and then solve the issue after.

Reply:
Ok I hope you will solve this issue as soon as possible